src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -486,6 +495,24 @@ private void ExpandPreviewControl()
             }
         }
 
+        /// <summary>
+        /// Sets ZIndex of node the maximum value.
+        /// </summary>
+        private void BringInFront()
+        {
+            if (IsMouseOver || PreviewControl.IsMouseOver || DynCmd.IsTestMode)
+            {
+                if (NodeViewModel.StaticZIndex == Int32.MaxValue)
+                {
+                    PrepareZIndex();
+                }
+                var index = ++NodeViewModel.StaticZIndex;
```

Comment:
I remember one of the test failed and you changed ++NodeViewModel.StaticZIndex  to  NodeViewModel.StaticZIndex++?